notebooks/vertex_genai/solutions/gemini_function_calling.ipynb

```diff
@@ -0,0 +1,1369 @@
+{
```

Comment:
<ul><li>Paste this cell in a pep8 formatter. Some of the lines are more than 80 chars</li><li>Really nice &amp; simple reasoning loop!</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/493/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Fixed thanks!</p>